oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -279,4 +279,17 @@ public void testGetCommandLine() {
 
         assertTrue(processesWithNonEmptyCmdLine >= 1);
     }
+
+    @Test
+    public void testConstructProcessWithGivenPid() {
+        SystemInfo si = new SystemInfo();
+        OperatingSystem os = si.getOperatingSystem();
+        OSProcess oldProcess = os.getProcess(os.getProcessId());
+        int givenPid = oldProcess.getProcessID();
```

Comment:
You get major bonus points for writing a testcase!

I'm curious why store the result of `getProcessID()` in a variable unless you want to test against it... why not just store `os.getProcessId()` two lines up and then use it twice?  (Or just not use a temp variable and only use `os.getProcessId()`.)

Reply:
Thanks :)

Good point, I change code to use temp variable twice (I think temporary variable is more meaningful and easier to read in this case).